kubectl-fdb/cmd/buggify_crash_loop.go

```diff
@@ -98,6 +112,72 @@ kubectl fdb -n default buggify crash-loop -c cluster pod-1 pod-2
 	return cmd
 }
 
+// updateCrashLoopContainerList updates the crash-loop container-list of the cluster
+func updateCrashLoopContainerList(kubeClient client.Client, clusterName string, containerName string, pods []string, namespace string, wait bool, clear bool, clean bool) error {
+	if len(containerName) == 0 {
+		return updateCrashLoopList(kubeClient, clusterName, pods, namespace, wait, clear, clean)
+	}
+	cluster, err := loadCluster(kubeClient, namespace, clusterName)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			return fmt.Errorf("could not get cluster: %s/%s", namespace, clusterName)
+		}
+		return err
+	}
+
+	processGroupIDs, err := getProcessGroupIDsFromPodName(cluster, pods)
+	if err != nil {
+		return err
+	}
+
+	patch := client.MergeFrom(cluster.DeepCopy())
+	if clean {
+		if wait {
+			if !confirmAction(fmt.Sprintf("Clearing crash-loop list for %s from cluster %s/%s", containerName, namespace, clusterName)) {
+				return fmt.Errorf("user aborted the removal")
+			}
+		}
+		containerIdx := 0
+		for _, crashLoopContainerObj := range cluster.Spec.Buggify.CrashLoopContainers {
+			if crashLoopContainerObj.ContainerName != containerName {
+				containerIdx++
+				continue
+			}
+			crashLoopContainerObj.Targets = nil
+			cluster.Spec.Buggify.CrashLoopContainers[containerIdx] = crashLoopContainerObj
+			break
+		}
+		return kubeClient.Patch(ctx.TODO(), cluster, patch)
+	}
+
+	if len(processGroupIDs) == 0 {
+		return fmt.Errorf("please provide atleast one pod")
+	}
+
+	if wait {
+		if clear {
+			if !confirmAction(fmt.Sprintf("Removing %v in %s from crash-loop container list of the cluster %s/%s", containerName, processGroupIDs, namespace, clusterName)) {
```

Comment:
```suggestion
			if !confirmAction(fmt.Sprintf("Removing %s in %s from crash-loop container list of the cluster %s/%s", containerName, processGroupIDs, namespace, clusterName)) {
```

I think that sentence needs some rephrasing e.g. for the container name `foundationdb` and the processGroupIDs `[storage-1, storage-2]` the message would be:


```
Removing foundationdb in [storage-1, storage-2] from crash-loop container list of the cluster ...
```